src/App.jsx

```diff
@@ -14,33 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'alphabet';
+const SORT_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const handleSort = (type, isReversed) => {
+  let sortedGoods = [...goodsFromServer];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  switch (type) {
+    case SORT_ALPHABET:
+      sortedGoods = [...sortedGoods].sort((goods1, goods2) =>
```

Comment:
```suggestion
      sortedGoods.sort((goods1, goods2) =>
```
You don't need to do it, as sort method mutate an array

Reply:
Okey, i understand